src/Illuminate/View/Compilers/ComponentTagCompiler.php

```diff
@@ -350,7 +350,7 @@ protected function parseBindAttributes(string $attributeString)
     protected function attributesToString(array $attributes)
     {
         return collect($attributes)
-                ->map(function ($value, string $attribute) {
+                ->map(function (string $value, string $attribute) {
```

Comment:
Why do we have scalar types here?

Reply:
I'm not really opposed to using them. I'm just not normally in the habit from so many years of them not being available.

Reply:
We could delete them here. The code will throw an exception if they are not castable to a string on the next line anyway.

Reply:
Let's do that in a separate pr :)